share/themes/style_dark_fusion.css

```diff
@@ -38,6 +38,12 @@ QToolButton[voice2="true"]:checked, QToolButton[voice2="true"]:pressed { color:
 QToolButton[voice3="true"]:checked, QToolButton[voice3="true"]:pressed { color: white; background: $voice3-bgcolor; }
 QToolButton[voice4="true"]:checked, QToolButton[voice4="true"]:pressed { color: white; background: $voice4-bgcolor; }
 
+/* ToolbarButton text to match ToolbarButton icons */
```

Comment:
Inconsistent spacing around braces

Reply:
OK - and also in the light_fusion. Thanks.